src/components/screens/send/index.js

```diff
@@ -19,9 +19,9 @@ const Send = ({ history }) => {
   const initialValue = parseSearchParams(history.location.search);
   const address = useSelector(state => state.account.info.LSK.address);
   const isAccountInitialized = useSelector(state => state.account.isAccountInitialised);
-  const isInitialization = initialValue.initialization;
+  const isReclaim = initialValue.reclaim;
 
-  if (isInitialization) {
```

Comment:
Reclaim transaction is not a transfer transaction. We should create a separate component instead of using the send component.